.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
 
 script:
   - ./gradlew check
-  - ./gradlew jacocoTestReport
+  - ./gradlew clean jacocoTestReport
```

Comment:
do we need this? it's removing the test results?

Reply:
perhaps you are right here. I will remove it. thanks

Reply:
```suggestion
  - ./gradlew jacocoTestReport
```